v2/src/constants.ts

```diff
@@ -48,6 +49,7 @@ export enum TagKeys {
 }
 
 export const runtimeLookup: { [key: string]: RuntimeType } = {
+  "dotnet6": RuntimeType.DOTNET,
```

Comment:
dotnet 8 is in a month, worth adding now? https://github.com/aws/aws-lambda-dotnet/issues/1611#issuecomment-1831523596

Reply:
I could add it, yet I cannot promise it will work.
On the same not, we'd have to add `Python_3_12` since its supposed to be released this month. I have no issues, for Python, since the definitions are already available in the CDK. Cannot say the same for `DOTNET_8`.

Reply:
As per our conversation, i'm going to close this and marked as resolved, will be addressed in separate PRs.